packages/relay/src/lib/eth.ts

```diff
@@ -1003,7 +1003,7 @@ export class EthImpl implements Eth {
       if (e instanceof JsonRpcError) {
         return e;
       }
-      return predefined.INTERNAL_ERROR();
+      return e.message ? predefined.INTERNAL_ERROR(e.message.toString()) : predefined.INTERNAL_ERROR();
```

Comment:
Also we only want to do this for error types we are aware of so we don't leak extra information no?

Reply:
Yes, so we are adding message for errors throws from the SDK. All information is coming from the error from SDK, so there should be any sensitive information, right ?